spec/models/tslr_claim_spec.rb

```diff
@@ -90,6 +90,39 @@
     end
   end
 
+  context "when saving in the “national-insurance-number” validation context" do
+    it "validates the presence of national_insurance_number" do
+      expect(TslrClaim.new).not_to be_valid(:"national-insurance-number")
+      expect(TslrClaim.new(national_insurance_number: "QQ123456C")).to be_valid(:"national-insurance-number")
+    end
+  end
+
+  context "when saving a record that has a National Insurance number" do
+    it "validates that the first two characters are letters" do
```

Comment:
It looks like these `it` blocks may have been iteratively written as the code was implemented in a TDD-style perhaps? That's 💯! That said, once the code is written and we have green, it can sometimes make sense to perform a refactor step on the tests too, as TDD can result in more tests than is strictly necessary. We want just enough to give us the confidence the code is doing what we want whilst keeping the number of tests to a minimum to avoid test bloat. Perhaps the refactor step here is to simplify this context down to a single `it "validates the National Insurance number is the correct format"` and bung a handful of valid and invalid NI numbers in there. Because the format for an NI number is established and well documented out on the internet, we don't necessarily need our tests to document it too.